src/assets/icon/socialMedia/InstagramIcon.tsx

```diff
@@ -0,0 +1,18 @@
+import { IconProp } from "../../../types/types";
+
```

Comment:
Sometimes you use 2 lines between imports and code. Sometimes only 1. Please keep one line or two in all places

Reply:
fixed